### DISCUSSION TITLE
【Feature request】 want to stop the effects of wind

### DISCUSSION THREAD

**Discussion by TojiroIwamoto** [pafuhana1213/KawaiiPhysics#138] [Ideas - 要望・アイデア]
I think we sometimes use alpha mainly to control the impact of physics, etc,
It seems that only the effect of wind is being reflected, so is it possible to use alpha to cut the effect of wind as well?

Or is it not a good idea to use it in this way?

I'll also put up an example code.

![image](https://github.com/user-attachments/assets/fa40017e-db84-417f-8988-002e76e66f2d)

**Comment by pafuhana1213** [comment]
@TojiroIwamoto さん、ありがとうございます！

Alphaが0のときはGetWindVelocity関数を間接的に呼ぶEvaluateSkeletalControl_AnyThreadがそもそも呼び出されないので、風の影響を受けないはずなのですが動いてしまっているということでしょうか…？
試しにサンプルプロジェクトでAlphaを0にしたのですが、KawaiiPhysicsの処理は風を含めて完全に止まってしまい…頂いた内容を再現することができませんでした…

もしよければ再現手順をいただけますと大変助かります！なお、既に試し済みかとは思いますが、風の影響をカットではなく補正したい場合はWindScaleプロパティをご利用頂ければと思います。

Thanks!

**Reply by TojiroIwamoto** [reply]
すいません！改めて再現を行って検証していたのですが、正常にαによる制御が効いていました！大変申し訳ありません！

少しカスタムがしてあって、添付画像の様なAnimNotifyからKawaiiPhysicsにαが流し込まれる作りになっているのですが、スクショの様にAnimationのプレビュー画面でアニメーションノティファイの再生フレームのマーカーが中途半端な位置にあるとPIE中の動作に影響してしまう挙動があって、これによる不具合を勘違いしてαが効いていない動作と誤認しちゃっていました……

完全に僕のミスぽい所で、KawaiiPhysicsの問題ではなさそうでした！
大変お恥ずかしいミスで申し訳ありません＞＜

ケースとしては、物理シミュレーションまで完全にベイクされたアニメーションと、ランタイムのKawaiiPhysicsのアニメーションをブレンドして切り替える様なケースだったので、αでKawaiiPhysics全体をカットする様な使い方としていました！
風のみカットする場合はWindscaleで調整する様にします！
ありがとうございます！

![image](https://github.com/user-attachments/assets/25cb0fed-74c4-4587-a92e-a30187c9dc8a)

**Comment by pafuhana1213** [comment]
いえいえ！手元の環境でご確認いただきありがとうございます！
もしまた何か問題やご要望などあればお気兼ねなくいただけますと幸いです！
